File: src/main/java/br/com/dgimenes/nasapicserver/model/APOD.java
package br.com.dgimenes.nasapicserver.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class APOD {

    @SerializedName("hdurl")
    private String url;

    @SerializedName("media_type")
    private String mediaType;

    private String explanation;

    private String title;

    public APOD(String url, String mediaType, String explanation, List<String> concepts, String title) {
        this.url = url;
        this.mediaType = mediaType;
        this.explanation = explanation;
        this.title = title;
    }

    public String getUrl() {
        return url;
    }

    public String getMediaType() {
        return mediaType;
    }

    public String getExplanation() {
        return explanation;
    }

    public String getTitle() {
        return title;
    }
}

File: src/main/java/br/com/dgimenes/nasapicserver/model/APOD.kt
package br.com.dgimenes.nasapicserver.model

import com.google.gson.annotations.SerializedName

data class APOD(@SerializedName("hdurl") val hdUrl: String?,
                val url: String,
                @SerializedName("media_type") val mediaType: String,
                val explanation: String,
                val title: String)
